contracts/ClaimsManager.sol

```diff
@@ -591,7 +587,13 @@ contract ClaimsManager is
         );
         ClaimState storage claimState = claimHashToState[claimHash];
         address arbitrator = claimState.arbitrator;
```

Comment:
The arbitrator of the dispute remains the same even if the dispute is resolved by an admin